tokio-trace/README.md

```diff
@@ -128,9 +128,25 @@ In order to record trace events, executables have to use a `Subscriber`
 implementation compatible with `tokio-trace`. A `Subscriber` implements a way of
 collecting trace data, such as by logging it to standard output.
 
-Unlike the `log` crate, `tokio-trace` does *not* use a global `Subscriber` which
-is initialized once. Instead, it follows the `tokio` pattern of executing code
-in a context. For example:
+There currently aren't too many subscribers to choose from. The best one to use right now
```

Comment:
The `lib.rs` top-level docs currently include most of the same content as `README.md`; it would be good to make this change there as well.